eyewear_shop/data/res_config_settings.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="res_config_settings_enable_eyewear_shop" model="res.config.settings">
+        <field name="enabled_extra_checkout_step" eval="1"/>
```

Comment:
As the dedicated extra page is added in demo, then this config should be enabled in demo only as well.